lib/components/contributors.jsx

```diff
@@ -91,7 +91,13 @@ const PlaceholderPhoto = styled(Flex) `
 
 const Contributors = (props) => {
   const theme = useTheme()
-  const list = sortBy(props.contributors, 'username')
+  let {
+    contributors
+  } = props
+  if (!props.minimalView) {
+    contributors = contributors.slice(0, 10)
+  }
+  const list = sortBy(contributors, 'username')
```

Comment:
Why are you sorting by username here? You are already sorting by contributions when the landr metadata is generated

Reply:
It was already there before, thought it was on purpose

Reply:
It can be removed :+1: